env_shell_python_git/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: swc-shell-python-git
+channels:
+  - swc
+dependencies:
+  - swc-shell
+  - swc-git
+  - swc-python
+  - python=2.7.*
```

Comment:
Thankfully, no pushing necessary.  It seems the website template needs to be updated to state that Python 3 will be used.